src/components/StateBrowserNetwork.vue

```diff
@@ -355,6 +410,17 @@ export default {
             this.channel_filter_display = !this.channel_filter_display;
             this.channel_add_display = false;
         },
+        toggleSection(type) {
+            if (type === 'channels') {
+                this.section_display_channels ?
```

Comment:
`this.section_display_channels = !this.section_display_channels;` :)

Reply:
Thanks, fixed that.